main.py

```diff
@@ -1,7 +1,44 @@
+from collections import namedtuple  # noqa: F401
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
The genre 'Dramma' seems to be a typo. It should be 'Drama' to match the standard genre name. This is corrected later in the script, but it's better to use the correct name initially to avoid unnecessary updates.